changelog/13842.contrib.rst

```diff
@@ -0,0 +1 @@
+Improve a few ``py`` tests.
```

Comment:
I think this changelog is a bit *too* trivial, I suggest removing it. I'll add skip-news label to the PR so that a changelog is not required.